.rubocop.yml

```diff
@@ -1,9 +1,11 @@
 require:
   - 'rubocop-capybara'
   - 'rubocop-factory_bot'
+  - 'rubocop-rspec_rails'
+
+plugins:
```

Comment:
gets rid of a deprecation warning that was added in 1.72.x.  i confirmed that this works in general in other projects by making this change first, and continuing to get rubocop rails errors before making the fixes rubocop suggested (see e.g. [this week's pres cat dep update PR](https://github.com/sul-dlss/preservation_catalog/pull/2385), where i made those changes as separate commits in order to better show that).